functions-app/school-results-load-cache/readme.md

```diff
@@ -0,0 +1,11 @@
+# TimerTrigger - JavaScript

```

Comment:
please replace MS default with info on the actual function

Reply:
done